.clang-tidy

```diff
@@ -8,7 +8,7 @@ Checks: >
   -misc-non-private-member-variables-in-classes,
   -misc-no-recursion,
   -misc-use-anonymous-namespace,
-  readability-identifier-naming'
```

Comment:
?

Reply:
There's a previous PR where I combined LLVM's and MLIR's clang-tidy styles and I messed it up with that straight quote.

Reply:
Oh! I see! For a second there I thought that line had been like that for a while and I was wondering how this was working at all 🤷